awx/ui/client/features/output/constants.js

```diff
@@ -16,7 +16,7 @@ export const JOB_STATUS_FINISHED = JOB_STATUS_COMPLETE.concat(JOB_STATUS_INCOMPL
 export const OUTPUT_ANSI_COLORMAP = {
     0: '#000',
     1: '#A00',
-    2: '#0A0',
+    2: '#080',
```

Comment:
The default ANSI green color we were using was too light, I've darkened it slightly.

![image](https://user-images.githubusercontent.com/214912/49091595-30721380-f22e-11e8-95df-e93756189f6b.png)